python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py

```diff
@@ -346,6 +354,30 @@ async def configure(self, config):
                 DeprecationWarning,
             )
 
+        self.time_on_target = config.time_on_target
+        if config.time_on_target == 0:
```

Comment:
if you agree to make `config.time_on_target = None` be the guard value, replace this by:

```
if config.time_on_target is None:
```

you can also do this in 1 line with:

```
self.time_on_target = self.compute_time_on_target() if config.time_on_target is None else config.time_on_target
```

Note it's dealing with `config.time_on_target = None` first, which I guess is the most common occurrence and hence, would make this more efficient.